src/chocolatey/infrastructure/powershell/PoshHostUserInterface.cs

```diff
@@ -129,7 +129,7 @@ public override void WriteProgress(long sourceId, ProgressRecord record)
             }
 
             // http://stackoverflow.com/a/888569/18475
-            Console.Write("\rProgress: {0}% - {1}".format_with(record.PercentComplete.to_string(), record.StatusDescription.PadRight(50, ' ')));
+            Console.Write("\rProgress: {0}% - {1}".format_with(record.PercentComplete.to_string(), record.StatusDescription).PadRight(Console.WindowWidth));

```

Comment:
Where does this pull console.windowwidth from? There are issues with redirects. As long as it is using the Console abstraction, it should work fine.